src/index.html

```diff
@@ -9,10 +9,83 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="/layout_stars/src/style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="frame--1">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="delimiter"></div>
+      <div class="stars stars--1">
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="delimiter"></div>
+      <div class="stars stars--2">
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+      </div>
```

Comment:
```suggestion
      <div class="stars stars--2">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```
rewrite it without the additional elements

Reply:
How i should indent them than?
(Its forbidden to use gap by task requirements)